src/service/direct.rs

```diff
@@ -51,7 +55,11 @@ impl ILnaddrService for DirectLnaddrService {
         Ok(Some(response))
     }
 
-    async fn get_destination(&self, domain: &str, username: &str) -> Result<Option<DestinationPaymentAddress>> {
+    async fn get_destination(
+        &self,
+        domain: &str,
+        username: &str,
+    ) -> Result<Option<DestinationPaymentAddress>> {
```

Comment:
nit: Why is this being re-formatted? Seems unrelated.

Reply:
Oh, I just noticed that this repo doesn't even have CI yet :see_no_evil: should add that and add a formatting check.

Reply:
I reverted all other formatting changes, those can be in a follow up.